Fits/Dreadnoughts/[FL33T] Anti-Cap Revelation Navy Issue.md

```diff
@@ -0,0 +1,52 @@
+# Revelation Navy Issue
+
+Anti-Capital RevNavy
+
+## Description
+
+Used to provide high DPS against other capital ships.  Highly ineffective for use against sub-capital targets.  Fuel bay should have 60k Helium Isotopes and be topped up with Strontium Clathrates. Fleet hangar should have an additional 1000 Strontium Clathrates.  The remainder of cargo space in both the fleet hangar and main hold should be filled with Cap Booster 3200s.
+
+## Fit
+```
+[Revelation Navy Issue, Anti-Cap Revelation Navy issue]
```

Comment:
To me, it feels odd to break with the other Revelation fitting and go with a capital cap inject here, when we can keep two heavies and T2 the plates. T2 plates on this look like a 1:1 upgrade in terms of cost <> efficiency, so we haven't yet hit diminishing returns?

Reply:
T2 Plates gain very little in terms of EHP for how expensive they are, even with Links + MG Amulets 2x T2 plates only add 240k EHP for 518mil extra cost. For the capital cap booster the rest of the fit should be exactly the same as the regular Rev though, nothing needed sacrificed for that iirc. And more cap is always going to be better on these things because of how much enemy neuting there is with RevNavy being the primary.